app/Traits/CachesOrders.php

```
@@ -0,0 +1,107 @@
+    {
+        return 'orders:version';
+    }
+
+    /**
+     * Get current global version for Orders cache keys.
+     */
+    public static function currentVersion(): int
+    {
+        $key = self::versionKey();
+
+        if (! Cache::has($key)) {
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Cache' in method 'currentVersion'. (undefined)

(StaticAccess)